static/styles/right-sidebar.scss

```diff
@@ -19,6 +19,14 @@
     overflow: auto;
 }
 
+/* Changes the color of right-sidebar scrollbar from the scrollbar used in
+rest of places */
+.simplebar-track {
+    .simplebar-scrollbar::before {
+        background-color: hsl(0, 0%, 38%);
```

Comment:
I think that changing the css in `static\styles\components.scss` should make this redundant.

Reply:
I have to change the color of the scrollbar only on the main app page. So, it is necessary to add these again. Changing it in `static\styles\components.scss` will change its color everywhere.

Reply:
Ah, okay. That makes sense if we want to retain the color for other places.